crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -721,6 +723,77 @@ impl KakarotTestEnvironmentContext {
         }
     }
 
+    pub async fn from_dump_state() -> Self {
+        // Construct a Starknet test sequencer
+        let sequencer = Arc::new(construct_kakarot_test_sequencer().await);
+
+        // Get root path
+        let workspace = std::process::Command::new(env!("CARGO"))
```

Comment:
is it possible to reuse the `root_project_path` macro defined in this file here?

Reply:
Oh didn't see that, will udpate!